admin_app/src/components/ContentModals.tsx

```diff
@@ -111,34 +118,67 @@ export const EditModal: React.FC<EditModalProps> = ({
             </h3>
             <XMarkIcon className="w-4 h-4 float-right" onClick={onClose} />
           </div>
-          <div className="relative p-2 flex flex-auto">
+          <div className="relative px-5 py-1 flex flex-auto mb-2">
             <input
               id="content_title"
               type="text"
-              className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-full dark:bg-gray-800 py-4 px-4 text-gray-800 dark:text-gray-400 overflow-auto required"
-              defaultValue={cardToEdit?.content_title}
+              className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-full dark:bg-gray-800 py-3 px-3 text-gray-800 dark:text-gray-400 overflow-auto required"
+              defaultValue={cardToEdit?.content_title || ""}
               placeholder="Enter content title"
               maxLength={150}
               onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
                 onTitleChange(e.target.value);
               }}
             />
           </div>
-          <div className="relative p-2 flex flex-auto">
+          <div className="relative px-5 py-1 flex flex-auto mb-2">
             <TextareaAutosize
               id="content_text"
               cols={75}
               maxRows={6}
               maxLength={1446} // Ensure len("*{title}*\n\n{text}") <= 1600
-              className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-full dark:bg-gray-800 py-4 px-4 text-gray-800 dark:text-gray-400 overflow-auto"
-              defaultValue={cardToEdit?.content_text}
+              className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-full dark:bg-gray-800 py-3 px-3 text-gray-800 dark:text-gray-400 overflow-auto required min-h-[10rem]"
+              defaultValue={cardToEdit?.content_text || ""}
               placeholder="Enter content text"
               onChange={(e: React.FormEvent<HTMLTextAreaElement>) => {
                 const target = e.target as HTMLTextAreaElement;
                 onContentChange(target.value);
               }}
             />
           </div>
+          <div className="relative px-5 py-1 flex flex-auto items-center">
+            <label
+              htmlFor="content_language"
+              className="block text-gray-700 text-sm font-bold mr-2"
+            >
+              Language
+            </label>
+            <div className="relative inline-block">
+              <select
+                id="content_language"
+                className="shadow appearance-none border active:outline-none border-neutral-400 text-sm rounded w-32 dark:bg-gray-800 py-2 px-3 text-gray-800 dark:text-gray-400 overflow-auto required mt-2"
+                defaultValue={cardToEdit?.content_language || Language.ENGLISH}
```

Comment:
Again `cardToEdit` is of type `ContentDataInEdit` which allows null language values.. But unlike title or text, I could make language value non-nullable for `ContentDataInEdit`. Not sure what would make more sense! @sidravi1